test/specs/modules/Search/Search-test.js

```diff
@@ -324,29 +290,11 @@ describe('Search', () => {
       openSearchResults()
       searchResultsIsOpen()
 
-      // arrow up
-      domEvent.keyDown(document, { key: 'ArrowUp' })
-
-      // selection moved to last item
-      wrapper
-        .find('SearchCategory')
-        .first()
-        .should.have.prop('active', false)
-
-      wrapper
-        .find('SearchResult')
-        .first()
-        .should.have.prop('active', false)
+      wrapper.should.have.state('selectedIndex', -1)
 
-      wrapper
-        .find('SearchCategory')
-        .at(categoryLength - 1)
-        .should.have.prop('active', true)
-
-      wrapper
-        .find('SearchResult')
-        .at((categoryLength * categoryResultsLength) - 1)
-        .should.have.prop('active', true)
+      // arrow up, selection moved to last item
+      domEvent.keyDown(document, { key: 'ArrowUp' })
+      wrapper.should.have.state('selectedIndex', (categoryLength * categoryResultsLength) - 1)
```

Comment:
I don't happy about this change, but it's impossible to test it now because children are not updated correctly.

Reply:
Reverted them,